src/sentry/issues/grouptype.py

```diff
@@ -207,10 +199,11 @@ class ProfileJSONDecodeType(GroupType):
     category = GroupCategory.PROFILE.value
 
 
-PROFILE_FILE_IO_ISSUE_TYPES = frozenset(
+PROFILE_ISSUE_TYPES = frozenset(
```

Comment:
You can use `get_group_types_by_category(GroupCategory.PERFORMANCE.value)` instead of this, I'm not sure why it exists.

Reply:
@wedamija put it in as a quick fix for `consumer` / `post_process`
https://github.com/getsentry/sentry/pull/45047

Reply:
I think his fix was too conservative, what we really want is to accept all types. Maybe checking the issue is in the profile category is not even useful?

Reply:
I think we should still have the issue in category check as a basic sanity check to avoid undefined types.